tests/playwright/sanity/core/base/document.test.ts

```diff
@@ -0,0 +1,54 @@
+import { expect, test } from '@playwright/test';
+import WpAdminPage from '../../../pages/wp-admin-page';
+
+test.describe( 'Document tests', async () => {
+	test( 'Converting Gutenberg page to sections columns',
+		async ( { page }, testInfo ) => {
+			const wpAdmin = new WpAdminPage( page, testInfo );
+			await wpAdmin.setExperiments( {
+				container: 'inactive',
+			} );
+
+			await wpAdmin.page.goto( '/wp-admin/post-new.php?post_type=page' );
+			await addElement( wpAdmin, 'list' );
+			await addElement( wpAdmin, 'heading' );
+			const editor = await wpAdmin.convertFromGutenberg();
+			const sections = await editor.getPreviewFrame().locator( '[data-element_type="section"]' ).count();
```

Comment:
You can cache `editor.getPreviewFrame()` in a variable as it is used multiple times.